client/starwhale/core/eval/cli.py

```diff
@@ -111,9 +111,9 @@ def _run(
     gencmd: bool,
     step: str,
     task_index: int,
-) -> None:
+) -> str:
     # TODO: tune so many arguments
-    JobTermView.run(
+    return JobTermView.run(
```

Comment:
should we drop the click client return value?

Reply:
done